addons/website_sale/static/src/js/website_sale.js

```diff
@@ -211,7 +211,7 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
             .data('combination');
 
         if (combination) {
-            return JSON.parse(combination);
```

Comment:
Only broken in 12.5 ?

Reply:
Haha i'm so used to it I was expecting a crash manager in previous versions...